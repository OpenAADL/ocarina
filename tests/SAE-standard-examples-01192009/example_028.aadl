-- test 11.1.1 Property Types draft-01192009

property set mine is
Length_Unit : type units ( mm, cm => mm * 10,
m => cm * 100, km => m * 1000 );
OnOff : type aadlboolean;
-- This type declaration references a separately declared units type
Car_Length : type aadlreal 1.5 .. 4.5 units mine::Length_Unit ;
-- This type declaration defines the units in place
Speed_Range : type range of aadlreal 0 .. 250 units ( kph );
Position : type record (
X: aadlinteger;
Y: aadlinteger; );
end mine;