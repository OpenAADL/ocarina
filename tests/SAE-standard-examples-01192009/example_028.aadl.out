example_028.aadl:8:28:  (signed aadlnumber)  is not a unit literal but  (real type)  is a unit type
example_028.aadl:8:35:  (signed aadlnumber)  is not a unit literal but  (real type)  is a unit type
Cannot analyze AADL specifications
